verticapy/plotting/_plotly/bar.py

```diff
@@ -39,18 +41,23 @@ def _compute_method(self) -> Literal["1D"]:
     # Styling Methods.
 
     def _init_style(self) -> None:
-        self.init_style = {}
+        self.init_trace_style = {"marker_color": PlotlyBase.get_colors(idx=0)}
```

Comment:
your class is inheriting from PlotlyBase, you should write self.get_colors

Reply:
done